content/docs/ref/runner.md

```diff
@@ -52,9 +52,10 @@ Any [generic option](/doc/ref) in addition to:
   purposes.
 - `--cloud-aws-security-group=<...>`:
   [AWS security group](https://docs.aws.amazon.com/vpc/latest/userguide/VPC_SecurityGroups.html)
-  name.
-- `--cloud-aws-subnet-id=<...>`:
-  [AWS subnet](https://docs.aws.amazon.com/vpc/latest/userguide/VPC_Subnets.html#subnet-basics).
+  name or ID.
```

Comment:
is this correct order of precedence?

Reply:
[Quite the opposite](https://github.com/iterative/terraform-provider-iterative/blob/e50ea7911a88de4229e1bb16c19fe8dd94aac48b/iterative/aws/provider.go#L182), but doesn't matter if we drop support for names, at least in the documentation. Moved to https://github.com/iterative/cml.dev/pull/185#discussion_r786990775.